pingouin/pairwise.py

```diff
@@ -630,11 +631,10 @@ def pairwise_tukey(data=None, dv=None, between=None, effsize='hedges'):
     >>> df = pg.read_dataset('penguins')
     >>> df.pairwise_tukey(dv='body_mass_g', between='species').round(3)
                A          B   mean(A)   mean(B)      diff      se       T  p-tukey  hedges
-    0     Adelie  Chinstrap  3700.662  3733.088   -32.426  67.512  -0.480    0.869  -0.070
-    1     Adelie     Gentoo  3700.662  5076.016 -1375.354  56.148 -24.495    0.001  -2.967
-    2  Chinstrap     Gentoo  3733.088  5076.016 -1342.928  69.857 -19.224    0.001  -2.894
+    0     Adelie  Chinstrap  3700.662  3733.088   -32.426  67.512  -0.480    0.881  -0.070
+    1     Adelie     Gentoo  3700.662  5076.016 -1375.354  56.148 -24.495   -0.000  -2.967
```

Comment:
@swallan why the negative sign here? a p-value cannot be negative.

Reply:
Absolutely right. I took a look at the parameters that are being used to get the p-values and evaluated them with mpmath, and it looks like the true result of what they should be is `mpf('0.0')`. (I spun up [this notebook](https://colab.research.google.com/drive/1PlgjaDbBKGl67QPOY5BDO_VDRp4bCpJ4?usp=sharing) to show this)

The result of evaluation with `scipy.stats.studentized_range` for those suspicious negative p-values are `-9.9253938401489e-14`. What we can do here is `np.clip(pvalues, 0, 1)`, to ensure they fall within the range while I submit a PR to scipy to include this behavior by default.

Reply:
This is great, thanks!

Reply:
Could you please update the docstring and then I think we'll be good to merge?

Reply:
Done! When [this pr](https://github.com/scipy/scipy/pull/15525) makes it into a SciPy release and pingouin's minimum SciPy version includes that feature I'll follow up to remove the unneeded clipping.

Reply:
@raphaelvallat Just wanted to check in -- is there anything else that you would like taken care of in this PR or is it merge ready?